src/Control/Monad/Freer/Reader.hs

```diff
@@ -73,6 +76,17 @@ asks f = f <$> ask
 runReader :: Eff (Reader e ': effs) a -> e -> Eff effs a
 runReader m e = handleRelay pure (\Reader k -> k e) m
 
+-- | `runReader` with the arguments flipped. May be more convenient for
+-- chaining `run*` commands together.
+--
+-- @
+-- eff :: (Member (Reader Env) effs, Member (Writer String) effs) => Eff effs ()
+--
+-- foo :: ((), String)
+-- foo = run . runReader env $ runWriter eff
+runReader' :: e -> Eff (Reader e ': effs) a -> Eff effs a
```

Comment:
`*With` would confuse normal usage of `With` in Haskell functions, which usually expect a function to add more control, a la `zip`/`zipWith`.

Or should I suck it up here, and just use `flip` directly in my own code?

```haskell
run . flip runReader env $ runWriter eff
```